### DISCUSSION TITLE
updating CMake minimum requirements?

### DISCUSSION THREAD

**Discussion by JorgeG94** [nlohmann/json#4306] [Q&A] [answered]
CMake says they'll deprecate compatibility with anything below CMake 3.5, nlohmann json currently has CMake minimum 3.1 which yields a warning:
```
CMake Deprecation Warning at external/json/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.
``` 
is there anything against changing the 3.1 to 3.5? 

Cheers

**Comment by nlohmann** [comment, marked as answer]
This should have been fixed with https://github.com/nlohmann/json/issues/4076.

**Reply by JorgeG94** [reply]
sorry you're right, my project was hardcoded to check out v3.11.2 :)
